From 4d18d63b6063783bd05df7904231b3e71b8d7359 Mon Sep 17 00:00:00 2001
From: Alex Kiernan <alex.kiernan@gmail.com>
Date: Fri, 1 Dec 2017 12:37:28 +0000
Subject: [PATCH] ostree-tmpfiles: Include ref changes

Pull in the reference changes for tmpfiles so ostree-prepare-root
works correctly

Signed-off-by: Alex Kiernan <alex.kiernan@gmail.com>
---
 src/boot/ostree-tmpfiles.conf | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/boot/ostree-tmpfiles.conf b/src/boot/ostree-tmpfiles.conf
index 49e2dcb..2ae7a8e 100644
--- a/src/boot/ostree-tmpfiles.conf
+++ b/src/boot/ostree-tmpfiles.conf
@@ -17,3 +17,9 @@
 
 # https://github.com/ostreedev/ostree/issues/393
 R! /var/tmp/ostree-unlock-ovl.*
+d /var/log/journal 0755 root root -
+L /var/home - - - - ../sysroot/home
+d /var/mnt 0755 root root -
+d /run/media 0755 root root -
+d /run/lock 1777 root root -
+d /var/lock/subsys 0755 root root -
